examples/mauna-loa/script.jl

```diff
@@ -0,0 +1,170 @@
+# # Mauna Loa time series example
+
+using CSV, DataFrames
+using AbstractGPs
+using ParameterHandling
+using Optim, Zygote
+using Plots
+
+# Let's load and visualize the dataset:
+
+begin
+    data = CSV.read("CO2_data.csv", Tables.matrix; header=0)
+    year = data[:, 1]
+    co2 = data[:, 2]
+
+    #Split the data into training and testing data
+    xtrain = year[year .< 2004]
+    ytrain = co2[year .< 2004]
+    xtest = year[year .>= 2004]
+    ytest = co2[year .>= 2004]
+end;
```

Comment:
`begin` doesn't introduce a new scope (which would also be undesired here):
```suggestion
data = CSV.read("CO2_data.csv", Tables.matrix; header=0)
year = data[:, 1]
co2 = data[:, 2]

#Split the data into training and testing data
xtrain = year[year .< 2004]
ytrain = co2[year .< 2004]
xtest = year[year .>= 2004]
ytest = co2[year .>= 2004]
```

Reply:
Yeah, that's left over from Pluto, but you're right in this format it's probably neater to do away with the begin-end. Just have to be careful so it doesn't end up printing all the arrays!